lldb/include/lldb/Core/Progress.h

```diff
@@ -99,6 +105,10 @@ class Progress {
 private:
   void ReportProgress();
   static std::atomic<uint64_t> g_id;
+  static std::atomic<uint64_t> g_refcount;
```

Comment:
By making the refcount static, it's shared across all the `Progress` instances, but IIUC, you want this refcount to be different for each progress category, right ? I actually don't even think the `Progress` object needs to know about its refcount, since only the code that will handle the progress event (in `Debugger::HandleProgressEvent` would need it).